modules/libcom/src/error/errlog.c

```diff
@@ -551,7 +573,13 @@ static void errlogInitPvt(void *arg)
     pvt.maxMsgSize = pconfig->maxMsgSize;
     ellInit(&pvt.listenerList);
     pvt.toConsole = TRUE;
+#ifdef vxWorks
+    pvt.console = fdopen(dup(ioTaskStdGet(0, 2)),"w");
+    pvt.closeConsole = 1;
+#else
     pvt.console = stderr;
+    pvt.closeConsole = 0;
```

Comment:
Setting `pvt.closeConsole` in the `#else` case shouldn't be necessary, right? Since nothing in the non-vxworks case writes into it.